src/bin/worker.js

```diff
@@ -239,6 +240,17 @@ program.parse(process.argv);
 
   await taskListener.connect();
 
+  // Set up the webhookServer instance for runtime.
+  var webhookServer = null;
+  try{
+    webhookServer = new WebhookServer(config.taskcluster);
+    let url = await webhookServer.start();
+    debug("webhookserver connected. hosted on "+url);
```

Comment:
Don't use `+` in `debug` statements... using `debug('bla bla: %s', url);` that way when `debug = function(){}` (empty function) which is the case when `DEBUG=''` it won't cost anything...